### ISSUE TITLE
Duplicate textual defintions

### ISSUE THREAD

**Issue by drseb** [obophenotype/human-phenotype-ontology#3750]
The travis build is failing because we have a lot of cases, where two distinct HPO classes have the same textual defintion. I hope you agree with me that it would be optimal to have no duplicated defintions.

Can somebody have a look at this? Maybe @nicolevasilevsky @LCCarmody @pnrobinson  ???

PROBLEM WITH DUPLICATE DEFINTION: an abnormality in which the nuclei of sarcomeres take on an abnormally central localization (or in which this feature is found in an increased proportion of muscle cells).
FOUND AS DEFINTION FOR TERMS: 
 - [x] Internally nucleated skeletal muscle fibers (HP:0031237)
 - [x] Centrally nucleated skeletal muscle fibers (HP:0003687)

PROBLEM WITH DUPLICATE DEFINTION: any anomaly of the function of the bronchi.
FOUND AS DEFINTION FOR TERMS: 
 - [x] Abnormal bronchus morphology (HP:0025426)
- [x] Abnormal bronchus physiology (HP:0025427)

PROBLEM WITH DUPLICATE DEFINTION: insufficient production of steroid hormones (primarily cortisol) by the adrenal glands as a result of a primary defect in the glands themselves.
FOUND AS DEFINTION FOR TERMS: 
 - [x] Adrenal insufficiency (HP:0000846)
- [x] Primary adrenal insufficiency (HP:0008207)

PROBLEM WITH DUPLICATE DEFINTION: uneven (irregular) increase in bone density of one or more of the phalanges of the hand.
FOUND AS DEFINTION FOR TERMS: 
- [x] Patchy sclerosis of finger phalanx (HP:0009772)
- [x] Patchy sclerosis of toe phalanx (HP:0010178)

PROBLEM WITH DUPLICATE DEFINTION: bullet-shaped phalanges refers to short and wide phalanges that taper distally . bullet-shaped phalanges lack the normal diaphyseal constriction.
FOUND AS DEFINTION FOR TERMS: 
- [x] Bullet-shaped phalanges of the 4th finger (HP:0009405)
- [x]  Bullet-shaped phalanges of the 3rd finger (HP:0009441)
- [x] Bullet-shaped phalanges of the 5th finger (HP:0009375)

PROBLEM WITH DUPLICATE DEFINTION: the epiphysis is normally located on the proximal surface of the phalanges. bracket epiphysis refers to an abnormality in which the epiphysis surrounds the phalanx, having a bracket-like form and reaching from the proximal side of the phalanx to the distal side.
FOUND AS DEFINTION FOR TERMS: 
  - [x]  Bracket epiphysis of the proximal phalanx of the thumb (HP:0009665)
  - [x] Bracket epiphysis of the distal phalanx of the 5th finger (HP:0009187)
  - [x] Bracket epiphysis of the distal phalanx of the 4th finger (HP:0009251)
  - [x] Bracket epiphysis of the proximal phalanx of the 3rd finger (HP:0009347)
  - [x] Bracket epiphysis of the middle phalanx of the 5th finger (HP:0009204)
  - [x] Bracket epiphysis of the distal phalanx of the 3rd finger (HP:0009336)
  - [x] Bracket epiphysis of the middle phalanx of the 3rd finger (HP:0009322)
  - [x] Bracket epiphysis of the distal phalanx of the thumb (HP:0009676)
  - [x] Bracket epiphysis of the proximal phalanx of the 5th finger (HP:0009197)
  - [x]  Bracket epiphysis of the proximal phalanx of the 4th finger (HP:0009262)
  - [x] Bracket epiphysis of the middle phalanx of the 4th finger (HP:0009215)
 - [x]  Bracket epiphysis of the distal phalanx of the 2nd finger (HP:0009503)

PROBLEM WITH DUPLICATE DEFINTION: digits are disproportionately narrow (reduced girth) for the hand/foot size or build of the individual.
FOUND AS DEFINTION FOR TERMS: 
 - [x]  Slender toe (HP:0011308)
  - [x]  Slender finger (HP:0001238)

PROBLEM WITH DUPLICATE DEFINTION: a type of strabismus in which the visual axis of one eye is higher than that of the other.
FOUND AS DEFINTION FOR TERMS: 
  - [x] Hypodeviation (HP:0025588)
  - [x]  Hyperdeviation (HP:0025587)

PROBLEM WITH DUPLICATE DEFINTION: clinical assay used to measure fibrinolysis. the euglobulin fraction of plasma is precipitated and used to form clot by addition of thrombin; after clot forms the rate of clot breakdown (fibrinolysis) can be monitored.
FOUND AS DEFINTION FOR TERMS: 
  - [x]  Prolonged euglobulin clot lysis time (HP:0040243)
  - [x] Reduced euglobulin clot lysis time (HP:0040247)

PROBLEM WITH DUPLICATE DEFINTION: eeg with sharp waves in the parietal region, i.e., sharp transient waves of a duration between 80 and 200 msec.
FOUND AS DEFINTION FOR TERMS: 
  - [x] EEG with occipital sharp waves (HP:0011292)
  - [x]  EEG with parietal sharp waves (HP:0011295)

PROBLEM WITH DUPLICATE DEFINTION: the cochlea is lacking the entire modiolus and cribriform area, resulting in a cystic appearance. this is accompanied by a large cystic vestibule.
FOUND AS DEFINTION FOR TERMS: 
 - [x] Incomplete partition of the cochlea type II (HP:0000376)
 - [x] Incomplete partition of the cochlea type I (HP:0011374)

PROBLEM WITH DUPLICATE DEFINTION: a pseudoepiphysis is a secondary ossification center distinct from the normal epiphysis.
FOUND AS DEFINTION FOR TERMS: 
 - [x] Pseudoepiphysis of the distal phalanx of the 5th finger (HP:0009188)
 - [x] Pseudoepiphysis of the middle phalanx of the 4th finger (HP:0009221)
   - [x]  Pseudoepiphysis of the distal phalanx of the 2nd finger (HP:0009509)
  - [x] Pseudoepiphysis of the distal phalanx of the 4th finger (HP:0009257)
  - [x]  Pseudoepiphysis of the proximal phalanx of the 3rd finger (HP:0009353)
  - [x]  Pseudoepiphyses of the 3rd finger (HP:0009417)
  - [x] Pseudoepiphyses of the 5th finger (HP:0009389)
 - [x]  Pseudoepiphysis of the proximal phalanx of the 5th finger (HP:0009200)
  - [x] Pseudoepiphysis of the middle phalanx of the 3rd finger (HP:0009328)
  - [x] Pseudoepiphysis of the proximal phalanx of the 4th finger (HP:0009268)
  - [x]  Pseudoepiphyses of the 2nd finger (HP:0009495)
 - [x]  Pseudoepiphyses of the 4th finger (HP:0009400)
  - [x]  Pseudoepiphysis of the middle phalanx of the 5th finger (HP:0009210)
  - [x]  Pseudoepiphyses of the phalanges of the hand (HP:0010235)
  - [x]  Pseudoepiphysis of the distal phalanx of the 3rd finger (HP:0009342)

PROBLEM WITH DUPLICATE DEFINTION: the presence of abnormal punctate (speckled, dot-like) calcifications in the epiphyses of phalanges of the fingers.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Epiphyseal stippling of finger phalanges (HP:0010237)
   - [x]  Epiphyseal stippling of toe phalanges (HP:0010171)

PROBLEM WITH DUPLICATE DEFINTION: partial or complete duplication of proximal phalanx of third toe.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Complete duplication of the proximal phalanx of the 3rd toe (HP:0100413)
   - [x]  Duplication of the proximal phalanx of the 3rd toe (HP:0100404)

PROBLEM WITH DUPLICATE DEFINTION: decreased concentration of dopamine in the cerebrospinal fluid (csf).
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Reduced CSF dopamine level (HP:0012656)
   - [x] Decreased CSF neopterin level (HP:0040205)

PROBLEM WITH DUPLICATE DEFINTION: abnormal arrangement of the structures of the axoneme, which is the cytoskeletal structure that forms the inner core of the motile cilium and displays a canonical 9 + 2 microtubular pattern of motile cilia studded with dynein arms.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Abnormal respiratory motile cilium morphology (HP:0005938)
   - [x]  Abnormal axonemal organization of respiratory motile cilia (HP:0012258)

PROBLEM WITH DUPLICATE DEFINTION: nail that appears thin when viewed on end.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Thin nail (HP:0001816)
   - [x]  Thin fingernail (HP:0012742)

PROBLEM WITH DUPLICATE DEFINTION: the gradual reduction in girth of the digit from proximal to distal.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Tapered toe (HP:0011309)
   - [x]  Tapered finger (HP:0001182)

PROBLEM WITH DUPLICATE DEFINTION: bracket epiphysis refers to an abnormality in which the epiphysis surrounds a phalangeal bone, having a bracket-like form and reaching from the proximal side of a phalanx to the distal side.
FOUND AS DEFINTION FOR TERMS: 
   - [x] Bracket epiphyses of the 2nd finger (HP:0009489)
   - [x]  Bracket epiphyses of the 4th finger (HP:0009394)
   - [x]  Bracket epiphyses of the 3rd finger (HP:0009411)
   - [x]  Bracket epiphyses of the phalanges of the hand (HP:0010229)
   - [x]  Bracket epiphyses of the 5th finger (HP:0009383)
   - [x]  Bracket epiphyses of the thumb (HP:0009687)

PROBLEM WITH DUPLICATE DEFINTION: in contrast to the proximal phalanges of the digits 2-5, the proximal phalanx of the thumb is embryologically equivalent to the middle phalanges of the other digits, whereas the first metacarpal is embryologically of phalangeal origin and as such equivalent to the proximal phalanges of the other digits.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Abnormality of the proximal phalanx of the thumb (HP:0009618)
   - [x]  Short 1st metacarpal (HP:0010034)
  - [x]  Aplasia of the 1st metacarpal (HP:0010035)

PROBLEM WITH DUPLICATE DEFINTION: collection of neutrophils in the epidermis.
FOUND AS DEFINTION FOR TERMS: 
  - [x]  Predominantly dermal neutrophilic infiltrate (HP:0031236)
   - [x]  Predominantly epidermal neutrophilic infiltrate (HP:0031235)

PROBLEM WITH DUPLICATE DEFINTION: hypospadias with location of the urethral meatus in the scrotum.
FOUND AS DEFINTION FOR TERMS: 
   - [x] Scrotal hypospadias (HP:0012853)
   - [x]  Midshaft hypospadias (HP:0012854)

PROBLEM WITH DUPLICATE DEFINTION: an elevated concentration of circulating cholesterol esters, which are fatty acid esters of cholesterol and make up about two-thirds of total plasma cholesterol.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  Increased cholesterol esters (HP:0500010)
   - [x]  Elevated cholesterol ester level (HP:0031211)

PROBLEM WITH DUPLICATE DEFINTION: the presence of an increased number of twists and turns of the retinal artery.
FOUND AS DEFINTION FOR TERMS: 
   - [x] Temporal artery tortuosity (HP:0030165)
   - [x]  Retinal arterial tortuosity (HP:0000631)

PROBLEM WITH DUPLICATE DEFINTION: eeg with focal sharp transient waves in the parietal region, i.e., focal sharp waves of a duration less than 80 msec followed by a slow wave.
FOUND AS DEFINTION FOR TERMS: 
   - [x]  EEG with parietal focal spike waves (HP:0012012)
   - [x]  EEG with temporal focal spike waves (HP:0012013)

**Comment by LCCarmody** [comment]
I'll work on these.  Not sure I will be able to get through them all.

**Comment by LCCarmody** [comment]
That's all I could do for now. Admittedly, I worked on the easier ones.  Some look like they should be merged, but didn't get a chance to look closely.  I can look at this a bit more later this afternoon.
@drseb @nicolevasilevsky @pnrobinson

**Comment by pnrobinson** [comment]
https://hpo.jax.org/app/browse/term/HP:0012013 is wrong
change "parietal" to "temporal" in the definition: EEG with focal sharp transient waves in the parietal region, i.e., focal sharp waves of a duration less than 80 msec followed by a slow wave.

**Comment by pnrobinson** [comment]
https://hpo.jax.org/app/browse/term/HP:0030165 is wrong
change "retinal" to "temporal" in the following definition: The presence of an increased number of twists and turns of the retinal artery.

**Comment by pnrobinson** [comment]
Change this definition to 
https://hpo.jax.org/app/browse/term/HP:0012854
Hypospadias with location of the urethral meatus in the middle of the inferior shaft of the penis
PMID: 12371222

**Comment by pnrobinson** [comment]
This is wrong
https://hpo.jax.org/app/browse/term/HP:0031236
change to Collection of neutrophils in the dermis.

**Comment by pnrobinson** [comment]
Increased cholesterol esters is a duplicate term -- I am merging

**Comment by pnrobinson** [comment]
I have resolve the remaining duplicated definitions
